x-pack/plugins/security_solution/public/timelines/components/edit_data_provider/index.tsx

```diff
@@ -248,6 +246,45 @@ export const StatefulEditDataProvider = React.memo<Props>(
           </EuiFlexItem>
 
           <EuiFlexItem grow={false}>
+            {showValueInput && (
+              <EuiFormRow label={i18n.VALUE_LABEL}>
+                <ControlledDefaultInput
+                  disableButtonCallback={disableButtonCallback}
+                  onChangeCallback={onValueChange}
+                  type={type}
+                  value={value}
+                />
+              </EuiFormRow>
+            )}
+
+            {showComboBoxInput && type !== DataProviderType.template && (
+              <EuiFormRow label={i18n.VALUE_LABEL}>
+                <ControlledComboboxInput
+                  disableButtonCallback={disableButtonCallback}
+                  onChangeCallback={onValueChange}
+                  type={type}
+                  value={value}
+                />
+              </EuiFormRow>
+            )}
+          </EuiFlexItem>
+
+          <EuiFlexItem grow={false}>
+            <EuiSpacer size="m" />
+          </EuiFlexItem>
+
+          <EuiFlexItem grow={false}>
+            {type === DataProviderType.template && showComboBoxInput && (
+              <>
+                <EuiCallOut
+                  color="warning"
+                  iconType="alert"
+                  size="s"
+                  title={`'${updatedOperator[0].label}' operator is unavailable with templates`}
```

Comment:
consider creating an `i18n` version of this `title`

Reply:
done!